Tests/SonarScanner.MSBuild.PreProcessor.Test/AnalysisConfigGeneratorTests.cs

```diff
@@ -384,6 +384,47 @@ public void GenerateFile_ExcludeCoverage_Exclusions_MultipleSpecified()
             .Which.Value.Should().Be("coverage1.xml,coverage2.xml,coverage3.xml,coverage4.xml");
     }
 
+    [DataTestMethod]
```

Comment:
I think you should also add some tests were the properties do not exist at all.
On all your tests everything exists and either has a value or is empty.